src/mammos_mumag/mesh.py

```diff
@@ -3,9 +3,10 @@
 import json
 import pathlib
 import shutil
-import time
 
-import requests
+from requests import Session
+from requests.adapters import HTTPAdapter
+from urllib3.util import Retry
```

Comment:
I would prefer not to have from-imports so that it is clear later on where an object is coming from.